### PR TITLE
CSS changes to make hint/guess text stand out against darker background

### LINKED ISSUE #38: Better readablility for solution texts
Especially green text on grey background is hard to read for my eyes (but that might be my color vision deficiency speaking). Anyhow, I'd suggest putting a background behind the text, or adding a text-shadow.
screenshot-2019-04-02_16:12:20

### PR DESCRIPTION
**Pull request by jaryncolbert**
Here's a shot at making the background darker to make the text more readable for hints and guesses. This should resolve #38, but eager to hear your feedback!

### CODE REVIEW

**Comment by jasonaowen** [comment]
cc @blinry - does that look more readable to you?

**Comment by jasonaowen** [comment]
@jaryncolbert this looks great! Thank you!

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,23 @@
 #filter option {
   background-color: #5cb85c;
 }
+
+
+/* Styling for blurring the text box surrounding the guesses and hints */
+form.hint {
+  padding-top: 10px;
+  padding-bottom: 10px;
```
[the remark this answers is not in the document]
Comment (jaryncolbert) [reply src/App.css]: I just committed 10a4851, which should fix both PX -> EM suggestions.

**Comment by jaryncolbert** [comment]
Thanks for the code review, @jasonaowen!

**Comment by jasonaowen** [comment]
And it's live on https://faces.recurse.com!

**Comment by jaryncolbert** [comment]
For a final recap, these commits: 
* Created translucent divs so that the text for hints and guesses stands out a bit more
* Changed the hint color from red to light blue
* Made the text size the same for hints, correct guesses, and incorrect guesses

Here are some screenshots for the final merged result: 

Screen Shot 2019-04-08 at 2 55 27 PM
Screen Shot 2019-04-08 at 2 56 01 PM

**Comment by blinry** [comment]
Thanks for working on this, @jaryncolbert, I love how this turne out!

**Pull request merged**